cli/src/lib.rs

```diff
@@ -2362,7 +2373,10 @@ fn deploy(cfg_override: &ConfigOverride, program_str: Option<String>) -> Result<
             );
             println!("Program path: {}...", binary_path);
 
-            let file = program.keypair_file()?;
+            let program_keypair_filepath = match program_keypair.clone() {
+                Some(program_keypair) => program_keypair,
```

Comment:
```suggestion
            let program_keypair_filepath = match program_keypair {
                Some(program_keypair) => program_keypair.clone(),
```
Almost nothing changed but I think better clone inner value than the whole Option.